subjects/apps/solution.js

```diff
@@ -95,8 +95,11 @@ const Chat = React.createClass({
   },
 
   componentDidUpdate() {
-    if (this.pinToBottom)
+    if (this.pinToBottom) {
       this.scrollToBottom()
+    } else {
+      this.refs.newNotifier.style = "display: block"
```

Comment:
One of the best things about React is that it lets me totally avoid resorting to CSS tricks like this one. If we don't need the `div.new-notifier` because we don't have any new messages, we can just omit it from the DOM entirely.